### ISSUE TITLE
Issue parsing pointers in external files

### ISSUE THREAD

**Issue by hilkeheremans** [BigstickCarpet/swagger-parser#35]
Hi there,

Great work on swagger-parser. I'm having an issue though: bundling pointers inside an externally reffed file seems to include the name of the pointer, where it shouldn't. I'm guessing the pointers are pretty much ignored.

Try parsing ```https://github.com/OAI/OpenAPI-Specification/tree/master/examples/v2.0/yaml/petstore-separate``` from the OpenAPI specs, and then validating it. 

File ```spec/swagger.api```refers to ```parameters.yaml#/tagsParam```. Bundling will include the ```parameters.yaml``` file and even select ```tagsParam```, but it also -- incorrectly -- includes the ```tagsParam```tag itself, rendering the resulting swagger.json invalid.

Let me know if I missed something here! Thanks for taking a look.

**Comment by hilkeheremans** [comment]
Just to add - if the issue can be verified - I would guess this issue belongs more in ```json-schema-ref-parser``` than here.

**Comment by BigstickCarpet** [comment]
Yeah, that's a bug that [has been fixed](https://github.com/BigstickCarpet/json-schema-ref-parser/commit/32510a38a29723fb24f56d30f055e7358acdd935) in the latest version of `json-schema-ref-parser`, but there are some breaking changes, so Swagger Parser [hasn't been updated yet](https://github.com/BigstickCarpet/swagger-parser/issues/30).

**Comment by hilkeheremans** [comment]
Ah, excellent! Thank you for putting in the time.

**Comment by BigstickCarpet** [comment]
Swagger Parser v4.0 _beta_ is now available, and includes the fix for this issue.
You can install the beta using `npm install swagger-parser@beta`.

- [v4.0 code branch](https://github.com/BigstickCarpet/swagger-parser/tree/releases/4.0.0)
- [v4.0 documentation](https://github.com/BigstickCarpet/swagger-parser/tree/releases/4.0.0/docs)
- [v4.0 changelog](https://github.com/BigstickCarpet/swagger-parser/blob/releases/4.0.0/CHANGELOG.md)
